packages/runtime/container-runtime-definitions/src/containerRuntime.ts

```diff
@@ -46,7 +44,6 @@ export interface IProvideContainerRuntime {
 }
 
 export interface IContainerRuntimeEvents extends IContainerRuntimeBaseEvents{
-    (event: "codeDetailsProposed", listener: (codeDetails: IFluidCodeDetails, proposal: IPendingProposal) => void);
```

Comment:
we need this event for versioning

Reply:
This change removes the event from the ContainerRuntime but leaves it on the Container.  Per our discussion today, I think it sounded like the ContainerRuntime would need to expose dirty state and a means to pause to complete the scenario, but that the Container event would be sufficient from the eventing side.

Reply:
until we have that solution in hand this is needed.